app/src/main/java/org/sopt/and/screen/LogInScreen.kt

```diff
@@ -0,0 +1,303 @@
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import kotlinx.coroutines.launch
+import org.sopt.and.MyActivity
+import org.sopt.and.R
+import org.sopt.and.SignUpActivity
+import org.sopt.and.component.TextField_Custom2
+
+@Composable
+fun LogInScreen(
+    modifier: Modifier = Modifier
+) {
+
+    val context = LocalContext.current
+
+    var id by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
+    var signUpId by remember { mutableStateOf("") }
+    var signUpPassword by remember { mutableStateOf("") }
+    var showPassword by remember { mutableStateOf(false) }
+
+    val launcher = rememberLauncherForActivityResult(
+        contract = ActivityResultContracts.StartActivityForResult()
+    ) { result ->
+        if (result.resultCode == Activity.RESULT_OK) {
+            signUpId = result.data?.getStringExtra("id") ?: ""
+            signUpPassword = result.data?.getStringExtra("password") ?: ""
+        }
+    }
+
+    val scope = rememberCoroutineScope()
+    val snackbarHostState = remember { SnackbarHostState() }
+
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        snackbarHost = {
+            SnackbarHost(hostState = snackbarHostState)
+        }
+    ) { innerPadding ->
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .fillMaxSize()
+                .background(color = colorResource(R.color.basic_background))
+                .padding(innerPadding)
+                .padding(10.dp)
+        ) {
+            Text(
+                text = "Wavve",
+                color = colorResource(R.color.white),
+                fontSize = 30.sp,
+                fontWeight = FontWeight.Bold
+            )
+
+            Spacer(Modifier.padding(40.dp))
+
+            TextField_Custom2(
+                id,
+                { text: String -> id = text },
+                "이메일 주소 또는 아이디",
+                true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                )
+            )
+
+            Spacer(Modifier.padding(5.dp))
+
+            TextField_Custom2(
+                password,
+                { text: String -> password = text },
+                "비밀번호",
+                showed = showPassword,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Done,
+                )
+            ) {
+                TextButton(
+                    onClick = {
+                        showPassword = !showPassword
+                    },
+                    content = {
+                        Text(
+                            text = "show", modifier = modifier.padding(7.dp)
+                        )
+                    }
+                )
```

Comment:
👍🏻 

```suggestion
                TextButton(
                    onClick = {
                        showPassword = !showPassword
                    }
                ) {
                       Text(
                            text = "show", modifier = modifier.padding(7.dp)
                        )
                }
```

이렇게 줄일 수 있습니다.

Reply:
람다를 잘 활용해보겠습니다!

Reply:
(참고) 가장 마지막에 오는 lambda는 trailing lambda(트레일링 람다)라고 불러요

Reply:
trailing lambda 이용하기...